crates/sui-types/src/lib.rs

```diff
@@ -134,3 +134,9 @@ impl MoveTypeTagTrait for u64 {
         TypeTag::U64
     }
 }
+
+impl MoveTypeTagTrait for ObjectID {
```

Comment:
It seems like you moved away from using `ObjectID` in a couple of places -- is this still necessary?